man/xml/daps-init.xml

```diff
@@ -6,6 +6,11 @@
   <!ENTITY daps_init "daps-init">
   <!ENTITY % entities SYSTEM "entity-decl.ent">
   %entities;
+  <!-- Needed for GitHub Action, see openSUSE/daps#585 -->
```

Comment:
#585 does not really explain this... this comment seems misleading

Reply:
Yeah. Maybe it's a better idea to use the pull request number instead of the issue. I've corrected that in commit 7183b653.